Update security/resiliency BBEs (#3795)
ballerina-platform/ballerina-distribution: examples/http-service-mutual-ssl/http_service_mutual_ssl.md

# HTTP service - Mutual SSL

> The `http:Listener` with enabled mutual SSL(mTLS) allows you to expose a connection secured with mutual SSL, which is a certificate-based authentication process in which two parties (the client and server) authenticate each other by verifying the digital certificates. It ensures that both parties are assured of each other's identity. The `http:Listener` secured with mutual SSL is created by providing the `secureSocket` configurations which require the word `require` as the `verifyClient`, the server's public certificate as the `certFile`, the server's private key as the `keyFile` and the client's certificate as the `cert`. Use this to secure the HTTP connection over mutual SSL.

Reviewer: ```suggestion
The `http:Listener` with enabled mutual SSL (mTLS) allows you to expose a connection secured with mutual SSL, which is a certificate-based authentication process in which two parties (the client and server) authenticate each other by verifying the digital certificates. It ensures that both parties are assured of each other's identity. The `http:Listener` secured with mutual SSL is created by providing the `secureSocket` configurations, which require the word `require` as the `verifyClient`, the server's public certificate as the `certFile`, the server's private key as the `keyFile` and the client's certificate as the `cert`. Use this to secure the HTTP connection over mutual SSL.
```

> The `http:Listener` with enabled mutual SSL (mTLS) allows you to expose a connection secured with mutual SSL, which is a certificate-based authentication process in which two parties (the client and server) authenticate each other by verifying the digital certificates. It ensures that both parties are assured of each other's identity. The `http:Listener` secured with mutual SSL is created by providing the `secureSocket` configurations, which require the word `require` as the `verifyClient`, the server's public certificate as the `certFile`, the server's private key as the `keyFile` and the client's certificate as the `cert`. Use this to secure the HTTP connection over mutual SSL.

Reviewer 2: ```suggestion
The `http:Listener` with mutual SSL (mTLS) enabled in it allows you to expose a connection secured with mutual SSL, which is a certificate-based authentication process in which two parties (the client and server) authenticate each other by verifying the digital certificates. It ensures that both parties are assured of each other's identity. The `http:Listener` secured with mutual SSL is created by providing the `secureSocket` configurations, which require the word `require` as the `verifyClient`, the server's public certificate as the `certFile`, the server's private key as the `keyFile`, and the client's certificate as the `cert`. Use this to secure the HTTP connection over mutual SSL.
```

> The `http:Listener` with enabled mutual SSL (mTLS) allows you to expose a connection secured with mutual SSL, which is a certificate-based authentication process in which two parties (the client and server) authenticate each other by verifying the digital certificates. It ensures that both parties are assured of each other's identity. The `http:Listener` secured with mutual SSL is created by providing the `secureSocket` configurations, which require the word `require` as the `verifyClient`, the server's public certificate as the `certFile`, the server's private key as the `keyFile` and the client's certificate as the `cert`. Use this to secure the HTTP connection over mutual SSL.

Reviewer: ```suggestion
The `http:Listener` with enabled mutual SSL (mTLS) allows exposing a connection secured with mutual SSL, which is a certificate-based authentication process in which two parties (the client and server) authenticate each other by verifying the digital certificates. It ensures that both parties are assured of each other's identity. The `http:Listener` secured with mutual SSL is created by providing the `secureSocket` configurations, which require the word `require` as the `verifyClient`, the server's public certificate as the `certFile`, the server's private key as the `keyFile` and the client's certificate as the `cert`. Use this to secure the HTTP connection over mutual SSL.
```
